partialWaveFit/fitResult.cc

```diff
@@ -519,9 +519,7 @@ double
 fitResult::intensity(const string& waveNamePattern) const
 {
 	vector<unsigned int> waveIndices = waveIndicesMatchingPattern(waveNamePattern);
-	if(waveIndices.size() == 1) {
-		return intensity(waveIndices[0]);
-	}
+
```

Comment:
Contrary to what I previously said, this appears correct to me now and should not be changed.